docs/src/content/docs/reference/clickup_tag_create.md

```
@@ -0,0 +1,42 @@
+---
+title: "clickup tag create"
+description: "Auto-generated reference for clickup tag create"
+---
+
+Create tags in a space
+
+### Synopsis
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)